packages/edit-site/src/components/block-editor/resizable-editor.js

```diff
@@ -57,20 +57,27 @@ function ResizableEditor( { enableResizing, settings, children, ...props } ) {
 				return;
 			}
 
-			let animationFrame = null;
+			let timeoutId = null;
 
 			function resizeHeight() {
-				if ( ! animationFrame ) {
-					// Throttle the updates on animation frame.
-					animationFrame = iframe.contentWindow.requestAnimationFrame(
-						() => {
-							setHeight(
-								iframe.contentDocument.documentElement
-									.scrollHeight
-							);
-							animationFrame = null;
+				if ( ! timeoutId ) {
+					// Throttle the updates on timeout.
+					timeoutId = iframe.contentWindow.setTimeout( () => {
+						const { readyState } = iframe.contentDocument;
+
+						// Continue deferring the timeout until the document is ready.
+						// Only then will it have a height.
+						if (
+							readyState !== 'interactive' &&
+							readyState !== 'complete'
+						) {
+							resizeHeight();
+							return;
 						}
-					);
+
+						setHeight( iframe.contentDocument.body.scrollHeight );
+						timeoutId = null;
+					}, 40 );
```

Comment:
Is `40` just an arbitrary delay here?

Reply:
It was 25 fps, but I've decided to change it to 30fps, which should be closer to `requestAnimationFrame`.